src/AzureSignTool/SignCommand.cs

```diff
@@ -96,7 +99,9 @@ internal sealed class SignCommand
         [Argument(0, "file", "The path to the file.")]
         public string[] Files { get; set; } = Array.Empty<string>();
 
+        private static DateTime _lastSigningOperation;
```

Comment:
Does this need to be static? Does it even need to be a field, or can it be part of the state in `Parallel.ForEach`?

Reply:
It's been a while since I wrote this and I'll have to double check whether throttling limits it you to a single request thread. I have a feeling I made it so that it did; if I did then there's not really any need for it to be static and I'll pull it in to the state.